client/state/concierge/shifts/reducer.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { stubTrue, stubFalse } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer, combineReducers } from 'state/utils';
+import { CONCIERGE_SHIFTS_FETCH, CONCIERGE_SHIFTS_UPDATE } from 'state/action-types';
+
+export const items = createReducer( null, {
+	[ CONCIERGE_SHIFTS_FETCH ]: () => null,
+	[ CONCIERGE_SHIFTS_UPDATE ]: ( state, { shifts } ) => shifts,
+} );
+
+export const isFetching = createReducer( false, {
```

Comment:
Since we're using the data-layer, we no longer need to use isFetching to prevent duplicate get calls. That said it's nice to use this for loading placeholders. I believe you can already infer this with your existing data structure. If items is `null` we should render a placeholder.

Unless there's anything missing we shouldn't need isFetching or the related selector.

Reply:
That sounds great. I'm always happy to chop down the state tree. It is now removed in d433e28.